en/docs/develop/product-apis/admin-apis/admin-v0.16/admin-v0.16.yaml

```diff
@@ -3600,6 +3641,7 @@ definitions:
       applicationId:
         type: string
         example: 01234567-0123-0123-0123-012345678901
+        readOnly: true
```

Comment:
Why do we need to set this property as read only. IIRC, this resource is not used to create applications.